openstudiocore/resources/gbxml/TestCube.xml

```diff
@@ -0,0 +1,6335 @@
+          <PlanarGeometry>
+            <PolyLoop>
+              <CartesianPoint>
+                <Coordinate>1</Coordinate>
+                <Coordinate>-0.000000000000003248601</Coordinate>
+                <Coordinate>0</Coordinate>
+              </CartesianPoint>
+              <CartesianPoint>
+                <Coordinate>1</Coordinate>
+                <Coordinate>1</Coordinate>
+                <Coordinate>0</Coordinate>
+              </CartesianPoint>
+              <CartesianPoint>
+                <Coordinate>1</Coordinate>
+                <Coordinate>1</Coordinate>
+                <Coordinate>1</Coordinate>
+              </CartesianPoint>
+              <CartesianPoint>
+                <Coordinate>1</Coordinate>
+                <Coordinate>-0.000000000000003248601</Coordinate>
+                <Coordinate>1</Coordinate>
+              </CartesianPoint>
+            </PolyLoop>
+          </PlanarGeometry>
+        </SpaceBoundary>
+        <SpaceBoundary isSecondLevelBoundary="false" surfaceIdRef="aim0939">
+          <PlanarGeometry>
+            <PolyLoop>
+              <CartesianPoint>
+                <Coordinate>0.00000000000000338396</Coordinate>
+                <Coordinate>1</Coordinate>
+                <Coordinate>0</Coordinate>
+              </CartesianPoint>
+              <CartesianPoint>
+                <Coordinate>0.00000000000000338396</Coordinate>
+                <Coordinate>1</Coordinate>
+                <Coordinate>1</Coordinate>
+              </CartesianPoint>
+              <CartesianPoint>
+                <Coordinate>1</Coordinate>
+                <Coordinate>1</Coordinate>
+                <Coordinate>1</Coordinate>
+              </CartesianPoint>
+              <CartesianPoint>
+                <Coordinate>1</Coordinate>
+                <Coordinate>1</Coordinate>
+                <Coordinate>0</Coordinate>
+              </CartesianPoint>
+            </PolyLoop>
+          </PlanarGeometry>
+        </SpaceBoundary>
+        <SpaceBoundary isSecondLevelBoundary="false" surfaceIdRef="aim0950">
+          <PlanarGeometry>
+            <PolyLoop>
+              <CartesianPoint>
+                <Coordinate>0.000000000000001928857</Coordinate>
+                <Coordinate>0</Coordinate>
+                <Coordinate>0</Coordinate>
+              </CartesianPoint>
+              <CartesianPoint>
+                <Coordinate>1</Coordinate>
+                <Coordinate>-0.000000000000003224601</Coordinate>
+                <Coordinate>0</Coordinate>
+              </CartesianPoint>
+              <CartesianPoint>
+                <Coordinate>1</Coordinate>
+                <Coordinate>-0.000000000000003224601</Coordinate>
+                <Coordinate>1</Coordinate>
+              </CartesianPoint>
+              <CartesianPoint>
+                <Coordinate>0.000000000000001766908</Coordinate>
+                <Coordinate>0</C
```

Comment:
@jasondegraw did you add these surfaceType attributes by hand?  Are there programs writing these?